src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -478,14 +498,12 @@ object DepositHandler {
     }
   }
 
-  def moveBagToStorage()(implicit settings: Settings, id: String): Try[File] =
+  def moveBagToStorage(depositDir: File, storageDir: File)(implicit settings: Settings): Try[File] =
     Try {
       log.debug("Moving bag to permanent storage")
-      val tempDir = new File(settings.tempDir, id)
```

Comment:
have you eliminated the `tempDir` from the settings?

Reply:
no, [it's still in use](https://github.com/DANS-KNAW/easy-sword2/pull/107/files#diff-1a93605ad9fd0ecb1bdfffe15af796acR89). However, `settings.tempDir` is actually the `depositDir` and `settings.depositRootDir` actually means `storageDir`. It is really confusing to just add a very simple thing to this project, given all the confusing names that are being used.

Some other remarks on a similar note:
* `storageDir` in fact is the `easy-ingest-flow-inbox`; not a directory in `easy-bag-store`!!!
* `id` is the name of deposit, a.k.a. the `bagId`, and probably a `UUID` (but I'm not sure about that!)

Reply:
You are right. Some names have been poorly chosen, a long time ago, when our design was still being developed. At the time we though that `easy-sword2` would store the deposit directly in the archival storage. I think it would be a good idea to do some name refactorings (could be in a separate pull request):

* `storageDir` -> `submittedDepositDir`
* `depositRootDir` -> `submittedDepositsRootDir`